py2puml/domain/umlclass.py

```diff
@@ -9,7 +9,15 @@ class UmlAttribute(object):
     type: str
     static: bool
 
+
+@dataclass
+class UmlMethod(object):
```

Comment:
remove `(object)` inheritance, this was required in Python 2 and is not necessary in Python 3